terragrunt/accounts/legacy/releases-dev/release-distribution/terragrunt.hcl

```diff
@@ -15,4 +15,7 @@ inputs = {
 
   static_bucket = "dev-static-rust-lang-org"
   log_bucket = "rust-release-logs"
+
+  static_cloudfront_weight = 0
```

Comment:
We may want to move this back to 50/50 or something, since otherwise we're not testing that CF is working on dev-static.

Reply:
I'm afraid that realistically the traffic volume against `dev-static` isn't high enough to have a lot of confidence whether both are working anyways. We'll have to test changes against both CDNs manually whenever we make them. But I'll drop this to 50/50 and to the same for `static.staging.crates.io`.

Reply:
Yeah, that's true. I think some amount of passive testing is likely though (e.g., when we announce a pre-release). That will help with changes upstream that aren't caused by us - things like the artifact size limitations we ran into previously - while being low-cost on our end.